language/02-guidelines/01-style/accessibility/index.md

```diff
@@ -32,6 +44,8 @@ When choosing colors and contrast ratios for typography it is important to adher
 
 Colors at a 50 value or lower are not accessible on neutral backgrounds, unless they're displayed at 18 pt or larger. There are no formal color contrast requirements for icons or glyphs that are paired with supporting text. If an icon is used separately from supporting text on a neutral background, use a color value of 50 or higher.
```

Comment:
The conversion is 1.333 (repeating) `px` per `pt`, which works out to about `23px` when printed, but it slightly different for screens.